src/components/Popups/LoginModal/LoginModal.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState, useEffect } from "react";
+import ModalWithForm from "../ModalWithForm/ModalWithForm.jsx";
+
+function LoginModal({
+  isOpen,
+  onClose,
+  onSubmit,
+  setSelectedPopup,
+  validationError,
+  setValidationError,
+}) {
+  const [data, setData] = useState({ email: "", password: "" });
+
+  const handleChange = (evt) => {
+    const { name, value } = evt.target;
+    setData((prevData) => ({
+      ...prevData,
+      [name]: value,
+    }));
+    setValidationError(false);
+  };
+
+  const _handleSubmit = (evt) => {
+    evt.preventDefault();
+    onSubmit(data);
+  };
+
+  const handleRedirect = () => {
+    onClose();
+    setSelectedPopup("signup-popup");
+  };
+
+  useEffect(() => {
+    setData({ email: "", password: "" });
+  }, [isOpen]);
+
+  return (
+    <ModalWithForm
+      title="Log in"
+      closePopup={onClose}
+      isOpen={isOpen}
+      onSubmit={_handleSubmit}
+      buttonText="Log in"
+      alternateOptionText="or Register"
+      alternateOptionHandler={handleRedirect}
+      validationError={validationError}
+      validationErrorText="The email or  password is incorrect"
+    >
+      <label className="modal__label">
+        Email
+        <input
+          className="modal__input"
+          id="email-login-id"
+          placeholder="Email"
+          type="email"
+          required
+          name="email"
+          value={data.email}
```

Comment:
Good job that you don't forget about `value` in the inputs if you have `onChange`